stl/inc/yvals_core.h

```diff
@@ -1082,7 +1083,7 @@
 #define __cpp_lib_constexpr_memory         201811L
 #define __cpp_lib_constexpr_numeric        201911L
 #define __cpp_lib_endian                   201907L
-#define __cpp_lib_erase_if                 201811L
+#define __cpp_lib_erase_if                 202002L

```

Comment:
`tests/std/tests/VSO_0157762_feature_test_macros/test.cpp` must be updated when changing feature-test macros.